median/plugin.go

```diff
@@ -95,19 +109,25 @@ type latestRoundRequested struct {
 func (c *chainReaderContract) LatestTransmissionDetails(ctx context.Context) (configDigest ocrtypes.ConfigDigest, epoch uint32, round uint8, latestAnswer *big.Int, latestTimestamp time.Time, err error) {
 	var resp latestTransmissionDetailsResponse
 
-	err = c.chainReader.GetLatestValue(ctx, c.contract, "LatestTransmissionDetails", nil, &resp)
-	if err != nil {
+	err = c.chainReader.GetLatestValue(ctx, contractName, "LatestTransmissionDetails", nil, &resp)
+	if err != nil && !errors.Is(err, types.ErrNotFound) {
```

Comment:
I still don't fully understand why is it fine for both `LatestTransmissionDetails` and `LatestRoundRequested` to not be found, we won't be immediately aware if onchain data for those two is not being transmitted. Can we at least log something here? How often do we expect for them to not be found

Reply:
If there's nothing transmitted yet, it gives nothing back.  It's consistent with today's behaviour without an error, but I can log a warning.